### ISSUE TITLE
Make scrollbar in chat list more visible

### ISSUE THREAD

**Issue by Rodriq** [RocketChat/Rocket.Chat#13892]
Currently what it looks like.
  One can hardly find it.
![screenshot-open rocket chat-2019 03 25-14-16-42](https://user-images.githubusercontent.com/39289592/54922857-545a1300-4f09-11e9-94a5-2480afd071dc.png)

**Comment by wreiske** [comment]
One of the biggest complaints I had coming in when moving to Rocket.Chat was that the scrollbar wasn't visible. Not everyone knows you can just scroll in the area without using the scrollbar. 

![image](https://user-images.githubusercontent.com/6295044/54931721-ee14c680-4eef-11e9-82a7-b13874d2c3e8.png)

```
*::-webkit-scrollbar-thumb {
    background-color: #F90;	
	background-image: -webkit-linear-gradient(45deg,rgba(255, 255, 255, .2) 25%,
											  transparent 25%,
											  transparent 50%,
											  rgba(255, 255, 255, .2) 50%,
											  rgba(255, 255, 255, .2) 75%,
											  transparent 75%,
											  transparent)
}
```
![image](https://user-images.githubusercontent.com/6295044/54931754-04bb1d80-4ef0-11e9-89ff-c187d3016e2d.png)

**Comment by Rodriq** [comment]
If a different color could be made available by default it'd be cool.
  The current one is almost not visible
I managed to change using webkit too
![Screenshot from 2019-03-25 17-33-06](https://user-images.githubusercontent.com/39289592/54937481-c55af400-4f24-11e9-8968-0d8da08578fc.png)

**Comment by rcadmin19** [comment]
In my experience webkit method in custom css does not work anymore in server version 3.13.2. Is there any new tricks for doing this in latest versions?

**Comment by rokkbert** [comment]
Seriously, the thin black scrollbar on dark grey, is such a ridiculous default. And it even fades out, so I have no chance to look for it with my eyes while I move the mouse to the general area. Can You please fix that to be visible even when the lighting conditions or the users eyes are less than perfect?
This feels like such an obvious case of "style over substance", "pretty dark theme design over usability". Does anybody consider rocketchat a work of art? For me it's a tool. It needs to be usable. What has the world come to...

**Comment by moell9** [comment]
Any update here ? I want to change the scrollbar color too.

**Comment by OrbitingOcelot** [comment]
Would love to see some improvements here, the usability of the chat list scrollbar is quite bad, very annoying in day to day work. Or, for people with visual/motor impairments, unusable.

**Comment by ans-artal** [comment]
+1
Very difficult to use the default color scrollbar.

Is there any way to customize the css only for my account ?

I don't a the rights to change "Administration > Workspace > Settings > Layout"

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Sorry, this error does not happen on the latest and maintained releases. It was fixed here: #32341

[Rocket.Chat](https://rocket.chat/) provides support for, at least, 3 months for each release and 6 months for the latest releases of each major version

I would suggest you update your Rocket.Chat version to the latest stable version, which you can find in [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases) and, check it [HERE](https://docs.rocket.chat/resources/get-support/enterprise-support#rocket.chat-versions) the table with versions and End of Life dates. 

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there
